src/github.com/getlantern/fronted/direct.go

```diff
@@ -180,6 +183,18 @@ func (d *direct) Dial(network, addr string) (net.Conn, error) {
 	}
 }
 
+func getCandidateCh() chan *Masquerade {
+	result, ok := _candidateCh.Get(5 * time.Minute)
+	if !ok {
+		panic("Unable to get candidateCh within 5 minutes")
+	}
+	return result.(chan *Masquerade)
+}
+
+func setCandidateCh(newCh chan *Masquerade) {
+	_candidateCh.Set(newCh)
+}
```

Comment:
A bit of a nit, but do we really need a separate method here @oxtoacart? I'm just thinking we probably only ever want to set it in `Configure`, but creating the separate method implicitly sort of encourages other code to call it.